utils/das_data_sorter.py

```diff
@@ -136,6 +137,53 @@ def gps_data(self, data:pd.Series) -> pd.Series:
         
         return new_gps_data
     
+    def smooths_data(self, data:pd.Series, technique:str, n:int=3) -> pd.Series:
+        if n < 3 or n > len(data):
+            raise ValueError("Number of smoothing points must be at least 3 and less than the length of the data set to perform smoothing.")
+        smooth_data_array = [] 
+
+        if technique == "mean" and n % 2 != 0: # Mean Smoothing for odd number N
+            for i in range(len(data) - N + 1):
+                data_points = data[i:i+N]
+                new_data_point = self.mean(data_points)
+                new_data_point = round(new_data_point, ndigits=2)
+                smooth_data_array.append(new_data_point)
+        elif technique == "mean" and n % 2 == 0: # Mean Smoothing for even number N
+            temp_array = []
+            for i in range(len(data) - N + 1):
```

Comment:
This seems like duplicated code as the odd pathway? Maybe we just need an if statement that checks if it's even so that it can run the for loop below?